discord/internal/cache.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+from typing import Any, OrderedDict
+from discord.types.snowflake import Snowflake
+from discord.user.user import User
+from discord.member.member import Member
+from discord.message.message import Message
+from discord.role.role import Role
+from discord.guild.guild import Guild
+from discord.api.handler import Handler
+

```

Comment:
Inside this folder there is a random .stats file. Not sure if that's supposed to be there?